docs/en/operation/Metrics-System.md

```diff
@@ -23,6 +23,10 @@ The following instances are currently supported:
 * Worker: The Alluxio worker process.
 * Client: Any process with the Alluxio client library.
 
+When the cluster is running with High Availability, by default the standby masters do not
+serve the metrics. From Alluxio 2.8 on, users can use `alluxio.standby.master.metrics.enabled=true`
```

Comment:
https://github.com/Alluxio/alluxio/blob/master/core/common/src/main/java/alluxio/conf/PropertyKey.java#L6853

```
    public static final String STANDBY_MASTER_METRICS_SINK_ENABLED =
        "alluxio.standby.master.metrics.sink.enabled";
    public static final String STANDBY_MASTER_WEB_ENABLED =
        "alluxio.standby.master.web.enabled";
```

From the code related, the `alluxio.standby.master.metrics.enabled=true` seems to be changed to `alluxio.standby.master.metrics.sink.enabled`

Reply:
ooops